src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -521,6 +523,37 @@ public void testPostOpenOrderInventoryUpdateOnlyForFirstPOL() throws Exception {
     verifyInventoryInteraction(resp, 1);
   }
 
+  @Test
+  public void testPutOrdersByIdTotalPiecesEqualsTotalQuantityWhenCreateInventoryIsFalse() throws Exception {
+    logger.info("=== Test Put Order By Id create Pieces when Item record does not exist ===");
+
+    CompositePurchaseOrder reqData = getMockDraftOrder().mapTo(CompositePurchaseOrder.class);
+    reqData.setId(ID_FOR_PRINT_MONOGRAPH_ORDER);
+    // Make sure that mock PO has 2 po lines
+    assertEquals(2, reqData.getCompositePoLines().size());
+
+    reqData.setWorkflowStatus(CompositePurchaseOrder.WorkflowStatus.OPEN);
+    verifyPut(String.format(COMPOSITE_ORDERS_BY_ID_PATH, reqData.getId()), JsonObject.mapFrom(reqData).toString(), "", 204);
```

Comment:
Please set `create_inventory` to false for one or both PO Lines so the test corresponds to its name

Reply:
Done